### PR TITLE
Minor edits to About page in preparation for V2

### PR DESCRIPTION
**Pull request by jvarness**
#127 has sub-bullets that mention the About page being refined.

I've added a little more information to this page that was missing. Please provide feedback if possible.

cc: @Stargator @kytrinyx

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-If you need to write a web/enterprise app, or you need data persistence or you need to do cryptography, no problem!
-Dart is all you require!
+ - Dart code can be transpiled into JavaScript for [web development](https://webdev.dartlang.org/), typically using [AngularDart](https://webdev.dartlang.org/angular).
+ - [Application servers](https://www.dartlang.org/dart-vm) can be written using Dart.
+ - Cross-platform mobile application development can be achieved through a popular framework called [Flutter](https://flutter.io/).
+ - [Google Firebase](https://github.com/firebase/firebase-dart) APIs can be integrated into Dart apps using the libraries written by the Firebase team.
 
 There is a rich ecosystem springing up around Dart because the standard (built-in) libraries can be supplemented by open source libraries found on [Pub](https://pub.dartlang.org).
-
-And the community has a voice in the development of the language since there are many standard tools/practices that extend the [Dart specification](https://www.dartlang.org/guides/language/spec), thanks to the [Dart Enhancement Proposals](https://github.com/dart-lang/dart_enhancement_proposals) (DEP)!
```
Comment (Stargator) [inline docs/ABOUT.md]: I would recommend keeping this as it's important to highlight the impact the community can have on the language's specification

**Comment by Stargator** [comment]
@jvarness Are you going to tackle the short description under the name as well in another PR?

File: docs/ABOUT.md (hunk 1 above)
Comment (jvarness) [reply docs/ABOUT.md]: The Dart Team is actually no longer accepting DEPs for community proposals:

https://github.com/dart-lang/dart_enhancement_proposals/blob/master/README.md

**Comment by jvarness** [comment]
> @jvarness Are you going to tackle the short description under the name as well in another PR? Per #127

@Stargator I was, but I'm actually not sure how that gets populated or what we need to include in the repo. Maybe @kytrinyx would be able to answer that?

File: docs/ABOUT.md (hunk 1 above)
Comment (Stargator) [reply docs/ABOUT.md]: Wow didn't hear that. That makes me sad.

**Comment by kytrinyx** [comment]
The blurb is a field in the `config.json`.

**Pull request merged**